optashift-employee-rostering-gwtui/src/main/resources/org/optaplanner/openshift/employeerostering/gwtui/client/skill/SkillListPanel.html

```diff
@@ -1,18 +1,33 @@
-<div>
-  <div style="padding-bottom: 10px">
-    <button id="refreshButton" class="btn btn-default" aria-label="Refresh">
-      <span class="glyphicon glyphicon-refresh" aria-hidden="true"></span> <span data-i18n-key="General.refresh">Refresh</span>
-    </button>
-  </div>
-  <table id="table"></table>
-  <div id="pagination"></div>
-  <div id="addForm" class="form-inline">
-    <div class="form-group">
-      <label for="skillNameTextBox" data-i18n-key="SkillListPanel.skillName">Skill name</label>
-      <input id="skillNameTextBox" type="text" class="form-control" placeholder="Driver's license">
-    </div>
-    <button id="addButton" class="btn btn-success" type="submit" aria-label="Add">
-      <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> <span data-i18n-key="General.add">Add</span>
-    </button>
+<div class="container-fluid">
+  <div class="row">
+        <div class="col-sm-12"> <!-- page-header ------->
+          <h1>Skills</h1>
+        </div>
   </div>
+  <div class="row toolbar-pf">
+
+        <div class="col-sm-12">
+          <div class="toolbar-pf-actions search-pf kie-search-pf has-button">
+            <div data-field="searchBar"></div>
+
+            <div class="toolbar-pf-action-right">
+              <div data-field="pager"></div>
+              <div class="form-group">
+                <button data-field="refreshButton" type="button" class="btn btn-default">Refresh</button>
+                <button type="button" data-field="addButton" class="btn btn-primary">Add Skill</button>
+              </div>
+            </div>
+          </div>
+        </div><!-- /col -->
+      </div>
+  <table data-field="table" class="table table-hover kie-tablelist">
+      <thead>
+        <tr>
+          <th colspan="1">Skill Name</th>
+          <th colspan="1"></th>
```

Comment:
If I'm not mistaken, `colspan="1"` is already the default behavior, so this can be removed.